src/Medication-9-0-7/Cert/PHR-Client/Dispense612Conversion/medmij-medication-dispense612conversion-phr-3-1b-requestdate.xml

```diff
@@ -10,14 +10,20 @@
     <test id="Scenario-3-1b">
         <name value="Scenario 3.1b"/>
         <description value="Filter op verstrekkingsperiode, alles vanaf T-90."/>
-        <nts:include value="phr-operation-search-dispense612conversion.xml">
+        <nts:include nts:in-targets="PHR-Client-Nictiz-Intern" value="phr-operation-search-dispense612conversion.xml">
```

Comment:
Dit zou alleen "Nictiz-Intern" moeten zijn.
Nb. in andere projecten noemen we dat "Nictiz-intern", met een kleine "i".